src/schema/objects/top_level_files.yaml

```diff
@@ -21,9 +21,18 @@ LICENSE:
 README:
   name: README
   description: |
-    In addition a free form text file (`README`) describing the dataset in more details SHOULD be
-    provided.
-    The `README` file MUST be either in ASCII or UTF-8 encoding.
+    A REQUIRED free-form text file, `README`, SHOULD describe the dataset in more detail.
+    The `README` file MUST be either in ASCII or UTF-8 encoding and MAY have one of the extensions:
+    `.md` ([Markdown](https://www.markdownguide.org/)),
+    `.rst` ([reStructuredText](https://docutils.sourceforge.io/rst.html)),
+    or `.txt` (unstructured text).
```

Comment:
just to make sure - the version with absent extension (as in current BIDS) corresponds to `.txt` (unstructured text) format.  I wonder there is a value from making it explicit -- probably not, but thought to mention. WDYT?

Reply:
From my experience, some software may not try to render a file without extension. Other than that, I personally don't see any advantage.

Reply:
Some people may write in markdown syntax in a `README` file, so I wouldn't make it seem like the lack of an extension somehow means that the contents should be unstructured.